docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
```

Comment:
It looks like `version` and `platform` keywords can be removed. I also had an issue connecting to port 5432 but I was able to connect when I made the port `5432` instead of `5432:5432`

Reply:
Were you able to get docker working?

Reply:
Yeah! Installed docker desktop and was able to run docker-compose with your file

Reply:
That's great! It seems like `version` and `platform` are redundant so I will remove that. I think it is still best to keep the port forwarding for `5432:5432` so the container knows what ports are exposed for Docker to use. This is more of a networking thing so it might be a network issue on your end.

Reply:
Yeah it was having issues because for me pgadmin had "ownership" of that port already